src/Query/ResultPrinters/ListResultPrinter.php

```diff
@@ -55,12 +55,17 @@ public function isDeferrable() {
 	 * @return string
 	 */
 	protected function getResultText( SMWQueryResult $queryResult, $outputMode ) {
-
 		$builder = $this->getBuilder( $queryResult );
 
 		$this->hasTemplates = $this->hasTemplates();
 
-		return $builder->getResultText() . $this->getFurtherResultsText( $queryResult, $outputMode );
+		$result = $builder->getResultText() . $this->getFurtherResultsText( $queryResult, $outputMode );
```

Comment:
What about just asking for `$queryResult->getCount()` here and not even calling the builder? 
The question is, what is expected for list/ol/ul for getCount === 0. Just an empty string or a wrapped empty string?

Reply:
`$queryResult->getCount()` is already compared to zero before this method is invoked. That just checks if there are no subjects though. The defaulting should also happen when there are subjects but no values.

If there are no values the default should be shown, at least if we want to restore the old behavior. Seems sensible enough to me.